e2e/terraform/compute.tf

```diff
@@ -4,6 +4,7 @@ resource "aws_instance" "server" {
   key_name               = module.keys.key_name
   vpc_security_group_ids = [aws_security_group.primary.id]
   count                  = var.server_count
+  availability_zone      = var.availability_zone
```

Comment:
Given that subnet is more specific than AZ, would it make sense to use the subnet `aws_subnet.default.id` here?  Assuming we are setting this for explicitness, the subnet is more explicit than relying on implicit AZ default subnet mapping.

Reply:
That's probably a good idea, but I think https://github.com/hashicorp/nomad/issues/7268 is more the direction we want to go anyways.